### DISCUSSION TITLE
密碼卡

### DISCUSSION THREAD

**Discussion by Mitochondrionjessica** [1101-GS4719-C/Python-Community-GS4719-C#1988] [General]
又到了要選下學期課程的時間了，由於我一直對找導師拿密碼卡才能選課感到百思不得其解，經查詢後的資料如下。
導師密碼卡機制已實施多年，秉持的理念：
「導師輔導選課」是一個「關懷」導生的過程，傳遞的不是一張張密碼卡而已，而是會面過程中，每一位導生感受到導師對他們的關心與鼓勵，帶給學生正面向上的能量，成為其生涯的助力及養分。
然而目前施行的過程中，有善用導師密碼卡制度的導師，以個別或小組或導聚方式親自交給導生並進行溝通，但也有交由助理、班代進行發送，這樣並沒有達到當初設立的理念，反而造成學生負面的觀感。

不管你認同還是不認同都要記得找導師拿密碼卡~
也歡迎大家發表你對拿密碼卡才能選課的想法~

**Comment by jylin34** [comment]
原來拿密碼卡是因為這種機制，今天第一次知道，以前沒有特別思考過拿密碼卡的意義，只覺得就是一個流程。然而我們班也都是教授轉交給班代再進行發放，所以可能也沒有達到當初這個機制所想達到的目的。

**Comment by cassy0201** [comment]
雖然知道這項機制是立意良善，但現實'並非能完全實際做到，自己系上就有一些教授是讓班代代為轉發，連面都沒見到，更不用說溝通、輔導了

**Comment by 710388** [comment]
拿的過程可以跟老師聊聊天也不錯，大學很少跟教授聊天的機會
